boards/arm/stm32mp157c_dk2/doc/stm32mp157_dk2.rst

```diff
@@ -192,8 +201,12 @@ Serial Port
 
 The STM32MP157C-DK2 Discovery board has 8 U(S)ARTs.
 The Zephyr console output is assigned by default to the ram console to be dumped
-by the Linux Remoteproc Framework on Cortex®-A7 core. The UART 4 can enabled as
-Cortex®-M4 console.
+by the Linux Remoteproc Framework on Cortex®-A7 core. The UART 4 can be enabled
```

Comment:
UART3 ?

Reply:
ST-Link provides Linux console using UART4. Theoretically, if the UART4 peripheral is not assigned to Linux console we can use it as Zephyr UART console. This is a very unlikely scenario.
Instead of promoting the usage of UART4, this part will be replaced by the usage of USART3 through GPIO

Reply:
Updated